### DISCUSSION TITLE
Webserver is slow after upgrading to v2.9.3

### DISCUSSION THREAD

**Discussion by csp33** [apache/airflow#41113] [General]
### Apache Airflow version

2.9.3

### What happened?

Webserver becomes really slow. However, once it loads, if I force-refresh the page, it reloads instantly (I guess that it's due to cache)
![image](https://github.com/user-attachments/assets/7c4f676b-a30b-451d-b696-ab4537f506bc)

### How to reproduce

-

### Operating System

Airflow in Kubernetes

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
It's not really actionable or reproducible,. Not sure what the screenshot means really and how it compares to previous version. No idea what deployment you have, no idea what else has changed - there are possibly proxies between you and the server there many things that might impact it and without more investigation what really is "slow" in your case and showing some logs, thsi is not an issue, it's at most a discussion - maybe someone will be able to help, but I strongly suggest trying to provide more details that might help someone to help you

**Comment by TreasureMaster** [comment]
I have the same problem after the update. I updated from version 2.6.2 to 2.9.3. I ran it with docker-compose and bitnami images. DEV cluster works well after the update. All containers are located on one server (master, scheduler, 3 workers, redis, postgres). LDAP authorization on a dedicated server. But RC cluster is very slow after the update. Main server contains master, scheduler and redis. Workers and postgres database are located on separate servers. LDAP authorization is the same.

**Comment by TreasureMaster** [comment]
Tried rolling back to 2.9.2. The problem still exists. Rollback to version 2.6.2 is not possible. Probably irreversible changes have occurred in the database

**Reply by phi-friday** [reply]
Since `airflow` manages the migration script with `alembic`, I think it should be possible to downgrade manually.

**Reply by TreasureMaster** [reply]
Yes, but I do not manage or have access to the database. The approval will take a lot of time.

**Comment by TreasureMaster** [comment]
After some testing, I assume that the problem may be in the database queries. The speed of query execution via API is the same as before. But accessing to index page `/home` is very slow. Login is also slow, but here a redirect to the index occurs. The other endpoints work fine. I'll wait for version 2.10. The authorization issue should be fixed. Some users cannot log in to the web interface.
